lib/rspec/rails/view_rendering.rb

```diff
@@ -81,7 +81,11 @@ def append_view_path(new_path)
       private
 
         def _path_decorator(path)
-          EmptyTemplateResolver.new(path)
+          if path.is_a?(::ActionView::Resolver)
+            path
```

Comment:
What we're trying to do is prevent the templates rendering, maybe this fix should be bypassing that check within empty template resolver?